Assets/MRTK/Providers/Experimental/WindowsMixedReality/WindowsMixedRealitySpatialAwarenessSceneUnderstandingObserver.cs

```diff
@@ -245,32 +348,31 @@ public TextAsset SerializedScene
             }
         }
 
+#endregion Profile
 
-        #endregion Profile
+#if SCENE_UNDERSTANDING_PRESENT
 
-        #region Private Fields
+#region Private Fields
```

Comment:
Just want to confirm: you don't like the ident here right?

Reply:
mistake. vs put the indent back on me